ViewLayer/BatteryUi/BatteryUi.cpp

```diff
@@ -0,0 +1,15 @@
+#include "BatteryUi.h"
+#include "ui_BatteryUi.h"
+BatteryUi::BatteryUi() :
+    ui_(new Ui::BatteryUi)
+{
+    ui_->setupUi(this);
+}
```

Comment:
Lets put a new line between the `}` of the previous function and the start of the next, for this and the other source files in this pr.